### ISSUE TITLE
Doesn't recognize that a song is playing

### ISSUE THREAD

**Issue by ProJosie** [kkevinm/iTunes-Discord-RP#72]
"Currently there is no song playing". Was fine and then this started showing

**Comment by Aviviviviveee** [comment]
yeah same problem for me (Issue #71), it works on my 2012 MBP however. Although since that is from 2012 it has the other issue where it requires you to have the RP as your latest opened window for it to not fall behind.

**Comment by JamesBissick** [comment]
https://github.com/NextFire/apple-music-discord-rpc
